BazelExtensions/xcodeproject.bzl

```diff
@@ -185,6 +195,14 @@ def _install_xcode_project_impl(ctx):
         "(rm -f $SRCROOT/external && ln -sf $PWD/../../external $SRCROOT/external)",
         'echo "' + output_proj + '" > ' + ctx.outputs.out.path,
     ]
+
+    if enable_indexing and relative_index_store_path:
+        command.extend([
+            "DD=$(xcodebuild -project " + output_proj + " -showBuildSettings 2>&1 | grep -e \"^\\s*BUILT_PRODUCTS_DIR = \" | cut -d'=' -f2 | xargs | sed -e 's/\\/Build.*//g')",
```

Comment:
Is it not possible to set this via `xcconfig` in Xcode?

Reply:
Done